postgresql-s3-backup/values.yaml

```diff
@@ -0,0 +1,47 @@
+# Default values for progresql-s3-backup.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+cronjob:
+  # At 01:00 every day
+  schedule: ""
+  annotations: {}
+  concurrencyPolicy: ""
+  failedJobsHistoryLimit: ""
+  successfulJobsHistoryLimit: ""
+
```

Comment:
Lets put default values for these.  failed: 3, success: 3, concurrencyPolicy: Forbid